xline/src/server/xline_server.rs

```diff
@@ -307,6 +310,7 @@ where
                 Arc::clone(&self.index_barrier),
                 Arc::clone(&self.id_barrier),
             ),
+            RocksSnapshotAllocator, // TODO: use allocator according to config. If user use rocksdb as storage,
```

Comment:
How come we hardcode `RockSnapshotAllocator` here if we use allocator according to a given config?

Reply:
FIxed.